lib/ViewModels/ChartPanelViewModel.js

```diff
@@ -90,7 +102,8 @@ var ChartPanelViewModel = function(options) {
             return [
                 new ChartViewModel(this, this, combineData(dataPairArrays), {
                     showRangeSelector: true,
-                    canDownload: true
+                    canDownload: true,
+                    height: this.height - 60
```

Comment:
Also is there a way to not hardcode this 60? Eg. if it's all due to the chart panel title bar, let's define that height somewhere and use that.